cli/pawls/preprocessors/tesseract.py

```diff
@@ -0,0 +1,104 @@
+from typing import List, Tuple, Dict
+import csv
+import io
+
+import pytesseract
+import pandas as pd
+from pdf2image import convert_from_path
+from PIL import Image
```

Comment:
If this is only for the type, I suggest dropping it - if not, can you add it to requirements.txt?

Reply:
Thanks! Just for type - OK I'll remove it!